### PR TITLE
Hardware mapping for motors

### PR DESCRIPTION
**Pull request by ghost**
Updated the hardware mapping for the motors.  The motor ID's are also set to match the PDP port numbers.  Note: Spark Max ID of 0 is not allowed, so Spark Max ID 16 is on physical port 0.

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -88,8 +90,8 @@ public Command GetAutonomousCommand () {
 
      // Button mappings
      private void ConfigureButtonBindings () {
-         new JoystickButton( mDriverXbox, 1).whenPressed( new InstantCommand( () -> mDrivetrain.SetHighGear( !mDrivetrain.IsHighGear() ), mDrivetrain ) );
-         new JoystickButton( mDriverXbox, 4).whenPressed( new InstantCommand( () -> mDrivetrain.SetReversed( !mDrivetrain.IsReversed() ), mDrivetrain ) );
+         //new JoystickButton( mDriverXbox, 1).whenPressed( new InstantCommand( () -> mDrivetrain.SetHighGear( !mDrivetrain.IsHighGear() ), mDrivetrain ) );
```
Comment (22nadsup) [inline src/main/java/frc/robot/RobotContainer.java:77]: why get rid of these?

Comment (ghost) [reply src/main/java/frc/robot/RobotContainer.java:77]: It's temporary.  The pneumatic hardware mappings weren't done.  That can be your task now :)

Comment (ghost) [reply src/main/java/frc/robot/RobotContainer.java:77]: Also, Blake needs to chime in on what buttons he wants to use for those functions.

**Review by DianaBellefeuille — APPROVED** [review]
Yay actual mappings

**Pull request merged**